code/__defines/dview.dm

```diff
@@ -1,14 +1,14 @@
 //DVIEW defines
 
 #define FOR_DVIEW(type, range, center, invis_flags) \
-	dview_mob.loc = center; \
+	dview_mob.forceMove(center); \
```

Comment:
Is there a reason behind this change? DVIEW mob is supposed to be unaffected by as many things as possible, so bypassing the Entered hooks might actually be worth it here.

Reply:
This should remain as direct loc setting, if only for performance reasons. This mob is moved around a LOT.